### ISSUE TITLE
Better coexistence with other Kodi addons using the Kodi databese

### ISSUE THREAD

**Issue by Larx** [jellyfin/jellyfin-kodi#427]
**Describe the bug**
As far as I understand it, the Jellyfin Kodi addon does some things with Kodi's database which are not "standard". In combination with other addons that seems to cause problems. E.g. the castagnit Netflix addon deletes all Jellyfin entries with every cleanup of the Kodi database. This is reproducable, but I experience strange database situations in other constellations, too ("watched" status suddenly lost, strange rescans of database, etc.). I think this problem of the addon is generally acknowledged.
This makes the Jellyfin addon unusable on any Kodi installation using not only Jellyfin media, but especially other streaming addons.

**To Reproduce**
Run Netflix addon in parallel with Jellyfin Addon.
After every library update by the Netflix addon the Jellyfin database entries are lost.

**Additional context**
The reason for this behaviour seems to be that Jellyfin's database entries do not really exist for Kodi. On every library "cleanup", they are marked as missing and deleted.
The Netflix addon creates "*.srtm" files for the additional library entries. So for Kodi the Netflix items are physically present on the harddisk and not deleted from the database.
I wonder if the Jellyfin addon could pursue a similar course. However, before any question arises, coding something like that is far beyond my personal skill :-).

**Comment by oddstr13** [comment]
Duplicate of #409
